dependencies.properties

```diff
@@ -34,8 +34,8 @@ version.io_grpc=1.32.2
 #   2) Replace all characters which are neither alphabetic nor digits with the underscore ('_') character
 maven.com_google_api_grpc_proto_google_common_protos=com.google.api.grpc:proto-google-common-protos:1.17.0
 maven.com_google_api_grpc_grpc_google_common_protos=com.google.api.grpc:grpc-google-common-protos:1.17.0
-maven.com_google_auth_google_auth_library_oauth2_http=com.google.auth:google-auth-library-oauth2-http:0.22.0
-maven.com_google_auth_google_auth_library_credentials=com.google.auth:google-auth-library-credentials:0.22.0
+maven.com_google_auth_google_auth_library_oauth2_http=com.google.auth:google-auth-library-oauth2-http:0.22.1
```

Comment:
This can't go in until this is published, which probably can't happen until Dec 1

Reply:
I see, please review other parts of the PR first, so we can get this PR in after google auth lib is published. Thanks!